tests/generation/FileGlobs.py

```diff
@@ -15,6 +15,7 @@ class TestQuestion(unittest.TestCase):
     def setUp(self):
         self.printer = ConsolePrinter()
         self.log_printer = LogPrinter(self.printer)
+        self.noninteractive = True
```

Comment:
Nope, this shouldn't be a default. Add a test that tests the noninteractive mode by making sure there aren't any questions asked.